dbt_copilot_helper/COMMANDS.md

```diff
@@ -589,6 +592,53 @@ copilot-helper domain assign-domain --app <app> --env <env> --svc <svc>
 - `--help <boolean>` _Defaults to False._
   - Show this message and exit.
 
+# copilot-helper svc
+
+[↩ Parent](#copilot-helper)
+
+    AWS Copilot svc actions with DBT extras.
+
+## Usage
+
+```
+copilot-helper svc deploy 
+```
+
+## Options
+
+- `--help <boolean>` _Defaults to False._
+  - Show this message and exit.
+
+## Commands
+
+- [`deploy` ↪](#copilot-helper-svc-deploy)
+
+# copilot-helper svc deploy
+
+[↩ Parent](#copilot-helper-svc)
+
+    Deploy image tag to a service, default to image tagged latest.
+
+## Usage
+
+```
+copilot-helper svc deploy --env <env> --name <name> --repository <repository> 
+                          [--image-tag <image_tag>] 
+```
+
+## Options
+
+- `--env <text>`
+
+- `--name <text>`
+
+- `--repository <text>`
+
+- `--image-tag <text>` _Defaults to latest._
```

Comment:
I went to --image-tag` over simply `--tag` as AWS Copilot uses the latter for other things and I wanted it to be distinct.